src/llmq/quorums_dkgsession.cpp

```diff
@@ -355,6 +371,10 @@ void CDKGSession::VerifyPendingContributions()
     for (const auto& idx : pend) {
         auto& m = members[idx];
         if (m->bad || m->weComplain) {
+            quorumDKGDebugManager->UpdateLocalMemberStatus(params.type, m->idx, [&](CDKGDebugMemberStatus& status) {
+                status.weComplain = true;
+                return true;
+            });
```

Comment:
In 8e4517f4:
Here we aren't complaining, we are skipping the bad and already complained quorum members. Should move this block to line 374/376.

Reply:
Yeah, line 393, when setting `CDKGMember::weComplain` :+1: